Core/include/Acts/EventData/TrackParametersConcept.hpp

```diff
@@ -26,24 +26,32 @@ using TypeParametersVector = typename T::ParametersVector;
 template <typename T>
 using TypeCovarianceMatrix = typename T::CovarianceMatrix;
 
-template <typename T>
-using ReturnTypeReferenceSurface =
-    decltype(std::declval<T>().referenceSurface());
 template <typename T>
 using ReturnTypeParameters = decltype(std::declval<T>().parameters());
 template <typename T>
 using ReturnTypeCovariance = decltype(std::declval<T>().covariance());
 template <typename T>
+using ReturnTypePosition4FromContext =
+    decltype(std::declval<T>().position4(GeometryContext()));
+template <typename T>
+using ReturnTypePosition4 = decltype(std::declval<T>().position4());
+template <typename T>
 using ReturnTypePositionFromContext =
     decltype(std::declval<T>().position(GeometryContext()));
```

Comment:
Same thing here, even though this is not part of the changeset.

Reply:
Fixed.